public/app/plugins/datasource/cloudwatch/metric-math/completion/statementPosition.ts

```diff
@@ -0,0 +1,21 @@
+import { LinkedToken } from '../../monarch/LinkedToken';
+import { StatementPosition } from '../../monarch/types';
+import { MetricMathTokenType } from './types';
+
+export function getStatementPosition(currentToken: LinkedToken | null): StatementPosition {
+  const previousNonWhiteSpace = currentToken?.getPreviousNonWhiteSpaceToken();
+
+  if (previousNonWhiteSpace?.is(MetricMathTokenType.Delimiter, ',')) {
+    return StatementPosition.SecondaryArgAfterPredefinedFunction;
+  }
+
+  if (previousNonWhiteSpace?.endsWith(')') && !previousNonWhiteSpace.is(MetricMathTokenType.Parenthesis, '()')) {
+    return StatementPosition.AfterFunction;
+  }
+
+  if (!currentToken || !currentToken.isString()) {
+    return StatementPosition.PredefinedFunction;
+  }
+
```

Comment:
Not completely necessary, but would be fairly straightforward to detect second and third argument for search expressions. If so, we could suggest [statistics](https://github.com/grafana/grafana/blob/sarahzinger/metric-math-expressions-revamp/public/app/plugins/datasource/cloudwatch/datasource.ts#L112) and periods. Up to you!

Reply:
I was able to suggest statistics, but I wasn't sure how to suggest a period of time, it seems to me like it'll just take any number so I'm not sure we can suggest that, but maybe there's a time related macro or something we should suggest? What do you think?

Reply:
It's any 60 seconds that can be multiplied by 60. And for a few high resolution metrics, it can be lower than that. But if we can suggest these number that would be sufficient in most scenarios I think. 
`10, 60, 300, 900, 3600, 21600, 86400`